### PR TITLE
Rwc2017 challenge final

### PR DESCRIPTION
**Pull request by Rayman**

### CODE REVIEW

File: challenge_final/launch/amigo_dependencies.launch [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```
Comment (MatthijsBurgh) [inline challenge_final/launch/amigo_dependencies.launch]: Not used in final, plus not a real feature at the moment, so could be removed in my opinion

File: challenge_final/scripts/challenge-final-old [hunk 2]
```diff
@@ -1,17 +1,10 @@
 #!/usr/bin/python
```
Comment (MatthijsBurgh) [inline challenge_final/scripts/challenge-final-old]: Should we keep the old challenges?

File: challenge_final/scripts/time.py [hunk 3]
```diff
@@ -39,7 +36,7 @@ def start_stopwatch(self):
     def get_stopwatch_time(self, stopwatch_number=0):
         if stopwatch_number >= len(self._stopwatches) or stopwatch_number < 0:
             now = datetime.now()
-            return now-now
+            return now - now
```
Comment (MatthijsBurgh) [inline challenge_final/scripts/time.py]: return now-now=zero? right??

File: challenge_final/src/challenge_final/final.py [hunk 4]
```diff
@@ -1,31 +1,21 @@
-# ROS
-import PyKDL as kdl
 import robot_smach_states as states
 import rospy
 import smach
-
-# TU/e Robotics
-from robocup_knowledge import knowledge_loader
+from challenge_storing_groceries.manipulate_machine import PlaceSingleItem
```
Comment (MatthijsBurgh) [inline challenge_final/src/challenge_final/final.py]: If used in other challenges, should it be moved to smach_states?

**Comment by MatthijsBurgh** [comment]
I am not sure what to do with: https://github.com/tue-robotics/tue_robocup/blob/rwc2017-challenge_final/challenge_final/src/challenge_final/pointing_detector.py#L50
It is unclear to me what code we used in the final

**Comment by Rayman** [comment]
We'll fix this later

**Pull request merged**